docs/api/desktop-capturer.md

```diff
@@ -12,24 +12,28 @@ title is `Electron`:
 // In the renderer process.
 const { desktopCapturer } = require('electron')
 
-desktopCapturer.getSources({ types: ['window', 'screen'] }, (error, sources) => {
+desktopCapturer.getSources({ types: ['window', 'screen'] }, async (error, sources) => {
   if (error) throw error
-  for (let i = 0; i < sources.length; ++i) {
-    if (sources[i].name === 'Electron') {
-      navigator.mediaDevices.getUserMedia({
-        audio: false,
-        video: {
-          mandatory: {
-            chromeMediaSource: 'desktop',
-            chromeMediaSourceId: sources[i].id,
-            minWidth: 1280,
-            maxWidth: 1280,
-            minHeight: 720,
-            maxHeight: 720
+  for (const source in sources) {
```

Comment:
Based on the PR title, I assume this should be `of` :)

```suggestion
  for (const source of sources) {
```

Reply:
Whoops!